com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -102,8 +102,9 @@ public partial class UniversalRenderPipelineAsset : RenderPipelineAsset, ISerial
         ScriptableRenderer[] m_Renderers = new ScriptableRenderer[1];
 
         // Default values set when a new UniversalRenderPipeline asset is created
-        [SerializeField] int k_AssetVersion = 5;
-        [SerializeField] int k_AssetPreviousVersion = 5;
+        const int k_MaxAssetVersion = 6;
+        [SerializeField] int k_AssetVersion = k_MaxAssetVersion;
+        [SerializeField] int k_AssetPreviousVersion = k_MaxAssetVersion;
```

Comment:
Shouldn't this be `k_MaxAssetVersion - 1`?

Reply:
This logic is rather confusing. Is k_MaxAssetVersion current pipeline asset version?

Reply:
It is very confusing indeed.